README.md

```diff
@@ -20,14 +20,16 @@ Please keep in mind that the public API is the `jesse.erl` module alone.
 
 You can fire up `jesse` from the CLI, with
 ```bash
-bin/jesse path_to_json_schema -- path_to_json_instance [path_to_json_instance]
+bin/jesse [path_to_json_schema] path_to_json_schema -- path_to_json_instance [path_to_json_instance]
```

Comment:
This might look a bit awkward but it is done that way for backwards compatibility.
If we switch to using [getopt](https://github.com/jcomellas/getopt) we might make it better... But probably there is little value in that here.